assets/js/modules/analytics-4/datastore/settings.test.js

```diff
@@ -304,6 +308,10 @@ describe( 'modules/analytics-4 settings', () => {
 						status: 200,
 						body: enabledSettingsMock,
 					} );
+					fetchMock.postOnce( settingsEndpoint, {
+						status: 200,
+						body: {},
+					} );
```

Comment:
This addition seems unnecessary. I've tried removing it and the tests still pass.

Reply:
Removed.